posts/2020-08-06-groovying-with-java.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Groovying with Java
+description: Learn how to leverage Groovy scripts in your Java code.
+canonical: https://dev.to/tomerfi/groovying-with-java-59hp
+date: 2020-08-06
+dir: ltr
+lang: en
+author:
+  name: Tomer Figenblat
+  jobTitle: Senior Software Engineer
+tags:
+- java
+- groovy
+- tutorial
+- code
+eleventyComputed:
+  coverImage: https://source.unsplash.com/WrYAR-yDwe8/640x426
+  coverImageAlt: a groovy image
```

Comment:
```suggestion
coverImage: https://source.unsplash.com/WrYAR-yDwe8/640x426
coverImageAlt: a groovy image
```
since they don't use `{{ data }}`, these can go in the root of the frontmatter